assets/sass/components/_badges-tags.scss

```diff
@@ -2,8 +2,9 @@
   .govuk-link {
     font-weight: normal;
   }
-
-  .govuk-tag--grey {
-    white-space: nowrap;
-  }
+}
+hr {
```

Comment:
This will update the styles for all `hr` tags - is that what you wanted to happen? I've force-pushed, so if you wanted to change this and make changes, you'd need to run `git fetch` and `git reset --hard origin/IC-1253-update-current-pages` to get the latest changes as the history has changed.

Reply:
yep this is the styling for the Hr tags site wide